### ISSUE TITLE
Missing support for labelFor property in javafx.scene.control.Label

### ISSUE THREAD

**Issue by gluon-bot** [gluonhq/scenebuilder#55]
Originally reported by: **Anonymous**

----------------------------------------

It seems that SceneBuilder has added support for the accessibility fields added recently (in JDK8_40?) but it still missing the support for the basic labelFor property of the javafx.scene.control.Label class.

This attribute can be set manually in the FXML by doing:

<Label labelFor=”$idOfAnotherNode”/>

Unfortunately SceneBuilder will erase those values if you happen to open the FXML later in it.

----------------------------------------
- Bitbucket: https://bitbucket.org/gluon-oss/scenebuilder/issue/55

**Comment by DesertCookie** [comment]
This issue still persists with Scene Builder 20.0.0. using Java and JavaFX 20.0.1.

**Comment by Locke** [comment]
As there were two points raised, it is not clear what "This issue still persists" from the previous comment refers to, so I'm writing this a bit more verbose.

I'm using the same version: JavaFX Scene Builder 20.0.0 with JavaFX Version: 20.0.1 on Java 20.0.1.

1) The labelFor attribute does not get removed for me, so that issues seems resolved or there is a more specific bug.

Given that [JDK-8090250](https://bugs.openjdk.java.net/browse/JDK-8090250) mentions that the order of elements matters, I have the following FXML, where I placed the TextField before the Label. When I open, modify, and save the file in SceneBuilder, the labelFor property remains.

```xml
<?xml version="1.0" encoding="UTF-8"?>

<?import javafx.scene.control.Label?>
<?import javafx.scene.control.TextField?>
<?import javafx.scene.layout.GridPane?>
<?import javafx.scene.layout.VBox?>

<VBox xmlns="http://javafx.com/javafx/20.0.1" xmlns:fx="http://javafx.com/fxml/1">
	<children>
		<GridPane>
			<children>
				<TextField fx:id="myCustomField" GridPane.columnIndex="1" />
				<Label labelFor="$myCustomField" text="My Custom Label" />
			</children>
		</GridPane>
	</children>
</VBox>
```

2) Support for the labelFor attribute in the UI (e.g. having a field in the properties / code panel) is still missing in the SceneBuilder version I'm using.
